examples/complete/main.tf

```diff
@@ -1,3 +1,82 @@
+############################################################################
+# Resource Group
+############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.4"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
 ##############################################################################
-# Complete example
+# Create Cloud Object Storage instance and a bucket
 ##############################################################################
+
+module "cos" {
+  source                 = "terraform-ibm-modules/cos/ibm"
+  version                = "7.4.3"
+  resource_group_id      = module.resource_group.resource_group_id
+  region                 = "us-south"
```

Comment:
Did you mean to hard code this or should it use the region passed in? It probably doesn't make a big difference, was thinking just for consistency of tests in case we get to point where we use a better region selector.

Reply:
good catch.